examples/threathunter/threat_intelligence/stix_taxii.py

```diff
@@ -26,8 +33,13 @@ def load_config_from_file():
     config_filename = os.path.join(os.path.dirname((os.path.abspath(__file__))), "config.yml")
     with open(config_filename, "r") as config_file:
         config_data = yaml.load(config_file, Loader=yaml.SafeLoader)
-        logging.info(f"loaded config data: {config_data}")
-        return config_data
+        config_data_without_none_vals = copy.deepcopy(config_data)
+        for site_name, site_config_dict in config_data['sites'].items():
```

Comment:
The reason this is needed is because you have empty config values in the file. The recommended practice would be to delete the config properties that are not needed. Though this code is fine to clean up ones as a defensive practice

Reply:
Would deleting the config properties be different than what we're doing here? Are you saying delete them from the file, and then load them into our config class?

Reply:
Yeah. If you delete the property of the config file then the yaml package won't load the property and it will be gone from the dictionary. However in this case since the properties are empty that is why they appear with None values

Reply:
Ah okay, that would be much cleaner. Do you know how complex it would be to do some file manipulation before loading the file? I'm reluctant to change the current working fix if modifying the file is complex and would extend the release time a lot

Reply:
So this really shouldn't be done with code. The intention is that the user should delete the properties they don't need. The code you have works, I just wanted to give some context to explain why this was occurring in case we wanted to update documentation any where

Reply:
Gotcha. Since the code is currently taking care of the None values, let's keep the documentation where it is.